pkg/apis/autoscaling/register.go

```diff
@@ -0,0 +1,20 @@
+// Copyright 2019 Google Inc. All Rights Reserved.
```

Comment:
nit as per #699 pull request and in other files.
```suggestion
// Copyright 2019 Google LLC All Rights Reserved.
```

Reply:
Ah - excellent catch. Will fix!

Reply:
Done!